homestar-runtime/src/event_handler/swarm_event.rs

```diff
@@ -93,6 +93,8 @@ async fn handle_swarm_event<THandlerErr: fmt::Debug + Send, DB: Database>(
     event_handler: &mut EventHandler<DB>,
 ) {
     match event {
+        // TODO: add identify for adding compatable kademlia nodes.
+        // TODO: use kademlia to discover new gossip nodes.
```

Comment:
this means rendezvous, right? I do know you're covering this in another branch nonetheless.

Reply:
alltogether yes. though it is very helpful to have identify along with rendezvous to check if it supports it (same with kademlia, gossipsub, request-response, ect.)